src/renderer/components/FormInputs/Toggle.tsx

```diff
@@ -10,15 +11,25 @@ type ToggleProps = {
   value: boolean;
   onChange: (val: boolean) => void;
   disabled?: boolean;
+  tooltipTitle?: string;
```

Comment:
```suggestion
  tooltip?: string;
```
Maybe just rename this to just `tooltip`.